custom_model_runner/datarobot_drum/drum/utils.py

```diff
@@ -171,3 +171,21 @@ def handle_missing_colnames(df):
         missing_lookup = {pycol: rcol for pycol, rcol in zip(missing_cols, r_vals)}
         return df.rename(columns=missing_lookup)
     return df
+
+
+def split_params_to_dict(params):
```

Comment:
Is this parsing content type header? lets use `from werkzeug.http import parse_options_header`

Reply:
Ah, its using command line params. I dont really see point of reinventing parser inside of parser, we can just make them separate parameters and dont parse anything manually

Reply:
it parses both: params from cli and params from content-type